x-pack/plugins/observability/public/components/app/section/ux/index.tsx

```diff
@@ -72,6 +97,15 @@ export function UXSection({ bucketSize }: Props) {
       }}
       hasError={status === FETCH_STATUS.FAILURE}
     >
+      <div style={{ height: 500 }}>
```

Comment:
```suggestion
      <div style={{ height: 320 }}>
```

Just changing the height for better proportions to the rest of the page.